src/test/resources/org/mapfish/print/config/sample_config_yaml/configWMTS.yaml

```diff
@@ -45,4 +45,3 @@ layouts:
           spacingAfter: 30
           width: 440
           height: 483
-      footer: *commonFooter
```

Comment:
Again, Why we removed the footer here?

Reply:
same as [here](https://github.com/geosolutions-it/mapfish-print-v2/pull/3#discussion_r1464358788)